ReSharper.FSharp/src/FSharp.Psi.Intentions/src/Intentions/UseNamedAccessAction.fs

```diff
@@ -17,7 +17,13 @@ type UseNamedAccessAction(dataProvider: FSharpContextActionDataProvider) =
     let mutable names = Array.empty
     let mutable tuplePat : ITuplePat = null
 
-    override x.Text = "Use named access"
+    override x.Text = 
+        let pattern = dataProvider.GetSelectedElement<IParametersOwnerPat>()
+        if isNull pattern then
+            "Use named fields in pattern"
```

Comment:
I think the text will only be accessed when `IsAvailable` returned `true`, so this pattern should never be `null` after that.

Reply:
Yeah wasn't sure about that part.